Sources/Afterpay/Wrappers/SwiftUIWrapper.swift

```diff
@@ -20,9 +20,12 @@ public extension View {
   ///   an Afterpay Checkout URL produced from the /checkouts endpoint. When a non nil URL is set,
   ///   a checkout sheet is presented. If the URL changes a new sheet is presented. If nil is set
   ///   the sheet is dismissed.
+  ///   - shouldLoadRedirectUrls: a boolean value that determines whether the redirect urls set
+  ///   when producing the checkout url should be redirected to. The default and recommended value is false
```

Comment:
```suggestion
  ///   when producing the checkout url should be allowed to load. The default and recommended value is false
```

Reply:
Is this clearer / more accurate? Or is it actually "allowed to start loading"? The intention here is to allow the request to send, right? That the server has given a response is primarily a verification of receipt; we don't actually need to load any part of the response. I wonder if we should clarify this to avoid giving the impression that the end user will actually view the page rendered at the redirect URL.

Reply:
Allow to start loading is probably more correct. I will update - thank you.

Reply:
Addressed in [573eba8](https://github.com/afterpay/sdk-ios/pull/236/commits/573eba8bf4a909e6c33f7daf1d39909a395a9d3d)